utils/build.ps1

```diff
@@ -1694,7 +1710,7 @@ function Build-CMark([Hashtable] $Platform) {
   Build-CMakeProject `
     -Src $SourceCache\cmark `
     -Bin (Get-CMarkBinaryCache $Platform) `
-    -InstallTo "$(Get-InstallDir $Platform)\Toolchains\$ProductVersion+$Variant\usr" `
+    -InstallTo "$(Get-InstallDir $Platform)\Toolchains\$ProductVersion+$PinnedToolchainVariant\usr" `
```

Comment:
This seems wrong - we shouldn't be using the `PinnedToolchainVariant`, but rather the current variant that we are building for as this is the distributed content.

Reply:
Re-using the libraries makes sense. However, I don't think that it makes sense to base this on the *PinnedToolchainVariant*. Once we distribute the NoAsserts toolchain, I could conceivably use the NoAsserts toolchain to build the early bits.